src/Ztm.Zcoin.Rpc/ZcoinRpcClient.cs

```diff
@@ -241,6 +277,62 @@ public Task<uint256> SendRawTransactionAsync(Transaction tx, CancellationToken c
             );
         }
 
+        public async Task<Transaction> SendAsync(
+            BitcoinAddress from,
+            BitcoinAddress to,
+            Property property,
+            PropertyAmount amount,
+            BitcoinAddress redeemAddress,
+            Money referenceAmount,
+            CancellationToken cancellation)
+        {
+            if (from == null)
+            {
+                throw new ArgumentNullException(nameof(from));
+            }
+
+            if (to == null)
+            {
+                throw new ArgumentNullException(nameof(to));
+            }
+
+            if (property == null)
+            {
+                throw new ArgumentNullException(nameof(property));
+            }
+
+            if (amount < PropertyAmount.One)
+            {
+                throw new ArgumentOutOfRangeException(nameof(amount), amount, "The value is less than one.");
+            }
+
+            var args = new List<object>()
+            {
+                from.ToString(),
+                to.ToString(),
+                ToNative(property.Id),
+                amount.ToString(property.Type),
+            };
+
+            if (redeemAddress != null) {
+                args.Add(redeemAddress.ToString());
+
+                if (referenceAmount != null)
+                {
+                    if (referenceAmount < Money.Zero)
+                    {
+                        throw new ArgumentOutOfRangeException(nameof(referenceAmount), amount, "The value is less than zero.");
+                    }
+                    args.Add(referenceAmount.ToString());
```

Comment:
on Zcoin side it will treat it as XZC unit and try to parse it to satoshi then we can pass amount string with or without period. Then better if we specify unit here.